sktime/forecasting/arch.py

```diff
@@ -0,0 +1,405 @@
+
+
+class ARCH(BaseForecaster):
+    """Directly interfaces ARCH models from python package arch.
+
+    Generalized Autoregressive Conditional Heteroskedasticity (GARCH)
+    model for forecasting votility in high frequency time series data.
+
+    ARCH models are a popular class of volatility models that use observed values
+    of returns as volatility shocks.
+
+    Implements Generalized Autoregressive Conditional Heteroskedasticity
+    (GARCH) models introduced and implemented in [1]_, [2]_, and [3]_ to
+    forecast the volatility in high frequency time series data, Implemented
+    using python package for ARCH models called "arch".
+
+    A complete ARCH model is divided into three components:
+        a mean model, e.g., a constant mean or an ARX;
+        a volatility process, e.g., a GARCH or an EGARCH process; and
+        a distribution for the standardized residuals.
+
+    Parameters
+    ----------
+    y : {np.ndarray, Series, None}
+        The dependent variable
+    x : {np.ndarray, DataFrame}, optional
+        Exogenous regressors.  Ignored if model does not permit exogenous
+        regressors.
+
+    Hyper-parameters
+    ----------------
+    mean : str, optional
+        Name of the mean model.  Currently supported options are: 'Constant',
+        'Zero', 'LS', 'AR', 'ARX', 'HAR' and  'HARX'
+    lags : int or list (int), optional
+        Either a scalar integer value indicating lag length or a list of
+        integers specifying lag locations.
+    vol : str, optional
+        Name of the volatility model.  Currently supported options are:
+        'GARCH' (default), 'ARCH', 'EGARCH', 'FIARCH' and 'HARCH'
+    p : int, optional
+        Lag order of the symmetric innovation
+    o : int, optional
+        Lag order of the asymmetric innovation
+    q : int, optional
+        Lag order of lagged volatility or equivalent
+    power : float, optional
+        Power to use with GARCH and related models
+    dist : int, optional
+        Name of the error distribution.  Currently supported options are:
+            * Normal: 'normal', 'gaussian' (default)
+            * Students's t: 't', 'studentst'
+            * Skewed Student's t: 'skewstudent', 'skewt'
+            * Generalized Error Distribution: 'ged', 'generalized error"
+    hold_back : int
+        Number of observations at the start of the sample to exclude when
+        estimating model parameters.  Used when comparing models with different
+        lag lengths to estimate on the common sample.
+    rescale : bool
+        Flag indicating whether to automatically rescale data if the scale
+        of the data is likely to produce convergence issues when estimating
+        model parameters. If False, the model is estimated on the data without
+        transformation.  If True, than y is rescaled and the new scale is
+        reported in the estimation results.
+
+    See Also
+    --------
+    Autoregressive Integrated Moving Average (ARIMA) models
```

Comment:
```suggestion
    ARIMA
    AutoARIMA
    StatsForecastAutoARIMA
```

Reply:
yes, that's the reason why readthedocs is failing. These need to be class names, and the reference section is auto-generated.

Reply:
Oh okay, I will change accordingly.